.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "powershell.codeFormatting.addWhitespaceAroundPipe": true
+}
```

Comment:
You need to add this file to your .gitignore file. Please remove this from the PR.

Reply:
I actually don't know why this shows  up in random branches. It's not consistent. I've added it to my gitignore now but it's still a curious thing.